opensearch/alegre_similarity.json

```diff
@@ -21,6 +16,14 @@
         "type": "knn_vector",
         "dimension": 768
       },
+    "vector_paraphrase-multilingual-mpnet-base-v2": {
```

Comment:
we are also adding new model,  was that the driver for this?

Reply:
This is simply bringing the file up-to-date with `elasticsearch/alegre_similarity.json`